xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -9,7 +9,9 @@
 #pragma once
 
 #include "XBDateTime.h"
+#include "addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_epg.h"
```

Comment:
Remove this line; not needed.

Reply:
I was surprised too since this odd line in pvr_recordings.h seems to required it.

https://github.com/xbmc/xbmc/blob/master/xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_recordings.h#L69

Reply:
Oh, but then we need to fix pvr_recordings.h. To avoid to include pvr_epg.h we should do:

`#define PVR_RECORDING_INVALID_SERIES_EPISODE -1`in pvr_recording.h

Can you add this change to this PR, please.

Reply:
Sure, I thought about that but figured that would be an API bump for PVR addons.

Reply:
Not really, as the numerical value stays the same. What changes is that you might save an include you currently need. No API bump needed. Would be different if you would need one more include due to our change. :-)

Reply:
OK I also changed the comment to match API 9 not sure if that impacts the API.